packages/rpms/SPECS/wazuh-server.spec

```diff
@@ -80,12 +80,13 @@ mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}var/lib/wazuh-server
 mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}usr/bin
 mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}var/log
 mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}usr/share/wazuh-server
+mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}usr/share/wazuh-server/bin
```

Comment:
Same here

Reply:
@RamosFe please, don't leave this comment unanswered and unresolved.

Reply:
Same case as https://github.com/wazuh/wazuh/pull/26912#discussion_r1853896660.